### PR TITLE
Rename period args to dateFrom/dateTo

### PR DESCRIPTION
**Pull request by Betree**
This is a breaking change for the host reports page, but since it's not been exposed publicly yet it's ok to take this shortcut.

### CODE REVIEW

File: server/graphql/v2/object/Host.js [hunk 1]
```diff
@@ -73,26 +73,36 @@ export const Host = new GraphQLObjectType({
...
+            description: 'The start date of the time series',
+          },
+          dateTo: {
+            type: GraphQLDateTime,
+            description: 'The end date of the time series',
           },
         },
         async resolve(host, args) {
-          const metrics = await host.getHostMetrics(args?.from, args?.to);
+          const metrics = await host.getHostMetrics(args.dateFrom || args.from, args.dateTo || args.to);
```
Comment (SudharakaP) [inline server/graphql/v2/object/Host.js:93]: Given that the arguments `from`, `to`, `dateFrom`, `dateTo` can be null; shouldn't we handle that? Either using the `?` or some other explicit handling? I am not quite understanding why you removed the `?` marks here. 🤔

**Review by SudharakaP — COMMENTED** [review]
@Betree : Everything looks good except for my small doubt below. 😉

File: server/graphql/v2/object/Host.js (hunk 1 above)
Comment (Betree) [reply server/graphql/v2/object/Host.js:93]: If all arguments are missing, `args` will just result in an empty object, it won't be nullish. The `?` were not needed here.

Comment (SudharakaP) [reply server/graphql/v2/object/Host.js:93]: Thanks for the explanation. 👍🏽

**Pull request merged**